#timesyncinfo
string devicename
string[] servers
int64[] update_count
float64[] time_updated
float64[] delay
float64[] offset
float64[] jitter
